test-poc/framework/src/main/java/org/keycloak/test/framework/server/NoAdminUserKeycloakTestServerConfig.java

```diff
@@ -0,0 +1,3 @@
+package org.keycloak.test.framework.server;
+
+public class NoAdminUserKeycloakTestServerConfig implements KeycloakTestServerConfig { }
```

Comment:
Maybe move this to base module instead of framework? It's a bit of a corner case thing, so probably doesn't need to be part of the framework itself